airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -592,6 +593,28 @@ def format_start_date(start_date: Optional[str]) -> Optional[str]:
             return pendulum.parse(start_date).strftime("%Y-%m-%dT%H:%M:%SZ")  # type: ignore[attr-defined,no-any-return]
         return None
 
+    def stream_slices(
+        self, *, sync_mode: SyncMode, cursor_field: List[str] = None, stream_state: Mapping[str, Any] = None
+    ) -> Iterable[Optional[Mapping[str, Any]]]:
+        base, start, end = (None, None, None)
+
+        now = pendulum.now(tz="UTC")
+        initial_date = pendulum.parse((stream_state or {}).get(self.cursor_field, self.start_date), tz="UTC")
+        period_end = initial_date.add(days=now.diff(initial_date).in_days())
+
+        slice_number = 1
+        while not end == now:
```

Comment:
This is not possible with current logic because `end` is equal to `None` when `now` is a datetime. So only `==` operation can be applied in this comparison